src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1479,28 +1547,217 @@ impl SimulationElement for A320BrakingForce {
     }
 }
 
-struct Door {
-    exit_id: String,
-    position: f64,
-    previous_position: f64,
+#[derive(PartialEq, Clone, Copy)]
+pub enum DoorControlState {
```

Comment:
**suggestion**: As this enum is currently unused outside of the module, it can remain private.

```suggestion
enum DoorControlState {
```